public/data/Horlakz.json

```diff
@@ -0,0 +1,54 @@
+{
+  "name": "Olakotan Daniel(Horlakz)",
```

Comment:
```suggestion
  "name": "Olakotan Daniel (Horlakz)",
```

Reply:
@kumarsonsoff3 I think he should remove this because it is automatically added from the file name.
```suggestion
  "name": "Olakotan Daniel",
```
![Screenshot from 2022-04-04 07-01-14](https://user-images.githubusercontent.com/87603425/161459768-af51a6ed-6fe8-4b1d-8e75-fe7e9c9b3374.png)

Reply:
@Shivansh-yadav13 You're right, I didn't notice this! 👍 

@Horlakz Please remove **"(Horlakz)"** from code, this'll be automatically added to your Linkfree Profile from the filename! 😄

Reply:
alright, then
I added it cos I want people to find me by my nick

Reply:
done